jbehave-support-core/src/main/java/org/jbehavesupport/core/rest/RestServiceHandler.java

```diff
@@ -100,6 +99,9 @@
     @Autowired
     private VerifierResolver verifierResolver;
 
+    @Autowired
+    private EqualsVerifier equalsVerifier;
```

Comment:
Constructor injection

Reply:
We dont want to do that - user extends the handler and could not use @RequiredArgsConstructor on his own handler if we would do that